src/configs/config.rs

```diff
@@ -0,0 +1,47 @@
+use clap::Parser;
+use config::{Config as ConfigLoader, File, FileFormat};
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct PostgresConfig {
+    pub url: String,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct Server {
+    // Which HOST the server is listening to
+    pub host: String,
+    // Which PORT the server is listening to
+    pub port: u16,
+    // Which PORT the server jaeger collection is listening to
+    pub jaeger_port: u16,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct Config {
+    // Which errors we want to log (info, warn or error)
```

Comment:
"Which errors we want to log (info, warn or error)" is not true for log_level

Reply:
Done